analysis/pseudobulk-bulk-prediction/model-notebooks/build-assess-models.Rmd

```diff
@@ -17,6 +19,12 @@ We build and assess the following random effects models for each project:
 * `fit_additive`: `bulk ~ pseudobulk + (1|sample_id)`
 * `fit_interaction`: `bulk ~ pseudobulk + (1|sample_id) + (1|sample_id:pseudobulk)`
 
+
+In addition, the notebook parameter `expr_threshold` determines how genes are filtered from the data before modeling: Only genes which are expressed in at least `2 * expr_threshold` in both modalities combined are retained.
```

Comment:
The wording here is tough, but I think it is worth trying to describe the intent rather than the precise calculation.

But we definitely want to say "greater than" rather than "at least" because filtering at 0 does filter samples!
```suggestion
In addition, the notebook parameter `expr_threshold` determines how genes are filtered from the data before modeling: Only genes which are expressed in greater than the proportion `expr_threshold` of samples, averaged across bulk and pseudobulk modalities, are retained.
```

Reply:
thank you! boy was I having a fun time phrasing this, which is why I gave up and just showed the math